tests/unit/pynwb_tests/test_core.py

```diff
@@ -207,6 +208,10 @@ def test_indexed_dynamic_table_region(self):
         dynamic_table_region = DynamicTableRegion('dtr', [0, 1], 'desc', table=table)
         assert dynamic_table_region[slice(0, 1)]
 
+    def test_nd_array_to_df(self):
+        col = VectorData(name='name', description='desc', data=np.array([[1, 1, 1], [2, 2, 2], [3, 3, 3]]))
+        DynamicTable('test', 'desc', np.arange(3, dtype='int'), (col, )).to_dataframe()
```

Comment:
Can you add assert statement checks to make sure the dataframe is converted appropriately?